draft-ietf-tls-tls13.md

```diff
@@ -2639,12 +2640,14 @@ and HelloRetryRequest are included in the transcript along with the
 new ClientHello.  For instance, if the client sends ClientHello1, its
 binder will be computed over:
 
-       ClientHello1[truncated]
+       Transcript-Hash(ClientHello1[truncated])
```

Comment:
You need to explicitly call out that because this is a *partial* ClientHello1, it doesn't need to be turned into the special message_hash form.